master/buildbot/steps/source/git.py

```diff
@@ -396,16 +390,30 @@ def _getSshDataPath(self):
     def _getSshPrivateKeyPath(self):
         return self.build.path_module.join(self._getSshDataPath(), 'ssh-key')
 
+    def _getSshWrapperScriptPath(self):
+        return self.build.path_module.join(self._getSshDataPath(), 'ssh-wrapper.sh')
+
+    def _getSshWrapperScript(self):
+        rel_key_path = self.build.path_module.relpath(
+                self._getSshPrivateKeyPath(), self._getSshDataWorkDir())
+
+        # note that this works on windows if using git with MINGW embedded.
```

Comment:
I think this is fine, but we should document this in the git step documentation. (not in the code :) )

Reply:
Fixed in daa2c095fe179de3bb15e912801ee0099a2e21e9.